tests/Doctrine/Tests/DBAL/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -2,13 +2,29 @@
 
 namespace Doctrine\Tests\DBAL\Functional\Schema;
 
-use Doctrine\DBAL\Schema;
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Index;
+use Doctrine\DBAL\Schema\Schema;
 use Doctrine\DBAL\Schema\Table;
 use Doctrine\DBAL\Types\Type;
 use Doctrine\Tests\TestUtil;
 
 class OracleSchemaManagerTest extends SchemaManagerFunctionalTestCase
 {
+    /**
+     * Number of SQL queries expected to be executed by createSchema.
+     *
+     * For Oracle, we expect 5 queries:
+     *   1. fetch sequences
+     *   2. fetch tables
+     *   3. fetch all columns
+     *   4. fetch all foreign keys
+     *   5. fetch all indexes
+     *
+     * @var int
+     */
+    protected $expectedQueryCountToCreateSchema = 5;
```

Comment:
It looks kind of okay but I still would prefer to not hard-code any concrete numbers. Instead, I'd create two schemas: for instance, one with one table, and one with two. And then made sure that the number of queries required to inspect them is the same.